src/esp/sim/ReplayBatchRenderer.cpp

```diff
@@ -79,9 +171,21 @@ ReplayBatchRenderer::ReplayBatchRenderer(
 
     renderer_->acquireGlContext();
   }
+
+  // Stuff?
+  for (int envIndex = 0; envIndex < config_.numEnvironments; envIndex++) {
+    auto& sensorMap = getEnvironmentSensors(envIndex);
+    CORRADE_INTERNAL_ASSERT(sensorMap.size() == 1);
+    for (auto& kv : sensorMap) {
+      CORRADE_INTERNAL_ASSERT(kv.second.get().isVisualSensor());
+      auto& visualSensor =
+          static_cast<esp::sensor::VisualSensor&>(kv.second.get());
+      renderer_->bindRenderTarget(visualSensor);
+    }
+  }
 }
 
-ReplayBatchRenderer::~ReplayBatchRenderer() {
+ReplayRenderer::~ReplayRenderer() {
   ESP_DEBUG() << "Deconstructing ReplayBatchRenderer";
```

Comment:
`"Deconstructing Replay~~Batch~~Renderer"`

Reply:
More like I should remove that useless debug print...